### ISSUE TITLE
The icon of the downloading file is not beautiful.

### ISSUE THREAD

**Issue by drc5521** [kiwibrowser/src.next#344]
**Is your feature request related to a problem? Please describe.**

The icon of the downloading file is not beautiful.
**Describe the solution you'd like**

The downloaded file, hope to have an icon display. Just like the windows file icon. as the picture shows.

**Additional context**

![pdf](https://user-images.githubusercontent.com/55195642/129729787-15c930e1-b7ca-4ff8-8024-e5f6502198af.png)
![word](https://user-images.githubusercontent.com/55195642/129729802-bded7fcd-2d80-4470-b09a-d3e05ae5a3b6.png)
![application](https://user-images.githubusercontent.com/55195642/129729806-1643a878-874f-44fe-a3a7-59af0cf5839e.png)
![ppt](https://user-images.githubusercontent.com/55195642/129729812-8db1475f-5c5a-4590-b2a0-ba2eceb3b004.png)

**Comment by gnadelwartz** [comment]
I agree it woud be easier to identity different file types with different icons, using windows or other copyrighted icons is not an option, but there should be opne source compatible iconsets exist. 

Google Chrome seems to us a simple solution: use the suffix of knonw files types as text on the icon:
![Screenshot 2021-08-18 08 11 29](https://user-images.githubusercontent.com/4593242/129847477-56a0c59f-a310-4bd7-832b-45a7923155ba.png)
